src/app/docs/page-summary/page-summary-docs.component.html

```diff
@@ -0,0 +1,22 @@
+<sky-docs-demo-page
+  moduleName="SkyPageSummarytModule"
+  moduleSourceCodePath="src/app/public/modules/page-summary/"
+  pageTitle="Page summary"
+>
+  <sky-docs-demo-page-summary>
+    The page summary component displays critical information and actions for users to access quickly and frequently.
+  </sky-docs-demo-page-summary>
+
```

Comment:
I'd consider adding a `Usage` section to the note at the top of the current page, minus the `Keep in mind` start. We could just make it a plain paragraph, or we could create a Use when and Don't use when items that present that content in our phase 2 paradigm.